src/tools/clippy/clippy_lints/src/unnamed_address.rs

```diff
@@ -96,9 +96,7 @@ impl LateLintPass<'_> for UnnamedAddress {
             if let ExprKind::Call(func, [ref _left, ref _right]) = expr.kind;
             if let ExprKind::Path(ref func_qpath) = func.kind;
             if let Some(def_id) = cx.qpath_res(func_qpath, func.hir_id).opt_def_id();
-            if match_def_path(cx, def_id, &paths::PTR_EQ) ||
-                match_def_path(cx, def_id, &paths::RC_PTR_EQ) ||
-                match_def_path(cx, def_id, &paths::ARC_PTR_EQ);
```

Comment:
AFAICT those paths were only used here. Could you also remove them from: 
[clippy_utils/src/paths.rs](https://github.com/rust-lang/rust-clippy/blob/0bca8dd2547fcc46796b53d338c0734575e51c72/clippy_utils/src/paths.rs#L18), please.

Reply:
Done !